tensorrt_llm/_torch/cute_dsl_utils.py

```
@@ -4,11 +4,24 @@
 IS_CUTLASS_DSL_AVAILABLE = False
 
+# Whether the public CuTe DSL package provides the SM107/Rubin helper module.
+# Rubin kernels stay disabled when it is absent and callers retain their
+# existing fallback paths.
+IS_CUTLASS_DSL_RUBIN_AVAILABLE = False
+
 if platform.system() != "Windows":
     try:
         import cutlass  # noqa
         import cutlass.cute as cute  # noqa
         logger.info(f"cutlass dsl is available")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)